src/neo-vm/ExecutionEngine.cs

```diff
@@ -281,11 +281,15 @@ private bool ExecuteInstruction()
                     {
                         return false;
                     }
-                case OpCode.THROWIF:
-                case OpCode.THROWIFNOT:
+                case OpCode.ASSERTIF:
+                case OpCode.ASSERTIFNOT:
```

Comment:
I think we need `ASSERT` only.

Reply:
@shargon Why do you think we need both `IF` and `IFNOT`?

Reply:
Only for save space, but it could be solved with one byte more. So i am not against this.

Reply:
in fact, it's not only one byte, at least you will need `DUP, NOP, ASSERT` 🤔

Reply:
Let's keep `ASSERT` only.

Reply:
Okay